### PR TITLE
5333: Added new components (isapi, isapi-sql) to helm charts

### PR DESCRIPTION
**Pull request by adrian-fitsios**

### CODE REVIEW

**Comment by stuart-bass-cgi** [comment]
Please update readme too.

File: local-helm/buyingcatalogue/templates/configmap.yaml [hunk 1]
```diff
@@ -1,5 +1,7 @@
 {{- $dapiUrl:= ternary (printf "%s.%s" (include "call-nested" (list . "dapi" "dapi.fullname")) .Release.Namespace) .Values.dapi.disabledUrl .Values.dapi.enabled -}}
 {{- $bapiUrl:= ternary (printf "%s.%s" (include "call-nested" (list . "bapi" "bapi.fullname")) .Release.Namespace) .Values.bapi.disabledUrl .Values.bapi.enabled -}}
+{{- $isapiUrl:= ternary "host.docker.internal" .Values.isapi.disabledUrl .Values.isapi.enabled -}}
```
Comment (stuart-bass-cgi) [inline local-helm/buyingcatalogue/templates/configmap.yaml]: host.docker.internal shouldn't be hardcoded here, but pulled out as a value

File: local-helm/local-overrides.yaml [hunk 2]
```diff
@@ -8,6 +8,7 @@
 #     repository: nhsd/buying-catalogue/document-dapi
 
 dbPassword: DisruptTheMarket1!
+clientSecret: TopSecretSecret
```
Comment (stuart-bass-cgi) [inline local-helm/local-overrides.yaml]: Can we please set to SampleCLientSecret to be consistent with the front end

File: local-helm/local-overrides.yaml [hunk 3]
```diff
@@ -8,6 +8,7 @@
 #     repository: nhsd/buying-catalogue/document-dapi
 
 dbPassword: DisruptTheMarket1!
```
Comment (stuart-bass-cgi) [inline local-helm/local-overrides.yaml:10]: Can both these top level value settings also go in buyingcatalogue/values.yaml  please

**Review by stuart-bass-bjss — CHANGES_REQUESTED** [review]

**Review by stuart-bass-bjss — APPROVED** [review]

**Pull request merged**